man/docker-pause.1.md

```diff
@@ -2,17 +2,17 @@
 % Docker Community
 % JUNE 2014
 # NAME
-docker-pause - Pause all processes within a container
+docker-pause - Pause all processes within one or more containers
 
 # SYNOPSIS
 **docker pause**
 CONTAINER [CONTAINER...]
 
 # DESCRIPTION
 
-The `docker pause` command suspends all processes in a container. On Linux,
-this uses the cgroups freezer. Traditionally, when suspending a process the
-`SIGSTOP` signal is used, which is observable by the process being suspended.
+The `docker pause` command suspends all processes within one or more containers.
```

Comment:
I'm not sure I like changing this in the description; it makes this sentence a lot more difficult to grasp. Overall I think I'm "good" with having "one or more ..." in the overall description, but we should not take it too far. Otherwise we end up with very difficult descriptions.

```Markdown
docker rm will remove one or more containers from the host node.
The container name(s) or ID(s) can be used. This does not remove
image(s). You cannot remove one or more running container(s)
unless you use the -f.
```

For the "one or more..", perhaps it would be even clearer if we used "the specified containers", e.g.

```Markdown
The `docker unpause` command un-suspends all processes in the specified container.
```

But that's out of scope for this PR